### PR TITLE
Add documentation on the connected GitHub app

### PR DESCRIPTION
**Pull request by wlonk**
four cats stretching

### CODE REVIEW

**Comment by wlonk** [comment]
[I want clearer documentation for setting up the necessary GitHub app(s).](https://trello.com/c/Lj4ZVeMF/153-i-want-clearer-documentation-for-setting-up-the-necessary-github-apps)

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -247,3 +247,28 @@ exposed to the front end, to properly generate translation files. See error
...
+-------------------------
+
+To deploy this app, you will need to set up a GitHub app and give it
+proper permissions. You can do that at
+``https://github.com/organizations/<your org>/settings/apps``
+
+The app will need the following permissions:
+
+ - Repository permissions
+    - Contents: read & write (maybe could be just read?)
```
Comment (davisagli) [inline CONTRIBUTING.rst]: needs write so we can commit captured changes, no?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -247,3 +247,28 @@ exposed to the front end, to properly generate translation files. See error
...
+    - Metadata: read-only
+    - Pull requests: read & write (maybe could be just read?)
```
Comment (davisagli) [inline CONTRIBUTING.rst]: needs write so we can create them, no?

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -247,3 +247,28 @@ exposed to the front end, to properly generate translation files. See error
...
+    - Commit statuses: read & write
+ - Organization permissions:
+    - None
+ - User permissions:
+    - None
```
Comment (davisagli) [inline CONTRIBUTING.rst]: Needs read for email address, no?

Comment (wlonk) [reply CONTRIBUTING.rst]: I recall setting that, but I can't see that in out set-up app. Let me look for it more.

Comment (wlonk) [reply CONTRIBUTING.rst]: Oh ugh it is there, it's just "Read-only" is the same width as "No access" and so my eyes kept glossing over it.

**Pull request merged**